tests/data/suppliermodule_test.py

```diff
@@ -10,3 +10,11 @@ def set_value(self, value):
 class DoNothing: pass
 
 class DoNothing2: pass
+
+class DoSomething:
+    def __init__(self, a_string: str, optional_int: int = None):
+        self.my_string = a_string
+        self.my_int = optional_int
+
+    def do_it(self, new_int: int) -> int:
+        return self.my_int + new_int
```

Comment:
This new class is for checking that #4551 works correctly with PlantUML output too.